backend/trivia_api/views.py

```diff
@@ -61,6 +61,71 @@ def users(user_id):
     return dummy_users
 
 
+@blueprint.route("/signup", methods=["GET", "POST"])
+def signup_post():
+    if request.method == "POST":
+        # return jsonify({"message": "success"}, 200)
+        data = request.get_json("password")
+        newusers = data["username"]
+        #  newusers2 = str(newusers)
+        newpasswords = data["password"]
+        passwords = generate_password_hash(newpasswords, method="sha256")
+        newuser = User(username=newusers)
+        newpass = User(password=passwords)
+        if (
+            User.query.filter_by(username=newuser.username).first() is None
+            and User.query.filter_by(password=newpass.password).first() is None
+        ):
+            entry = User(username=newuser.username, password=newpass.password)
```

Comment:
```suggestion
            new_user = User(username=username, password=generate_password_hash(password, method="sha256"))
```

variable naming/logic fix